libs/execution_context/ebpf_maps.c

```diff
@@ -618,52 +623,102 @@ _update_array_map_entry_with_handle(
     uintptr_t value_handle,
     ebpf_map_option_t option)
 {
-    if (!map || !key || (option == EBPF_NOEXIST))
+    ebpf_result_t result = EBPF_SUCCESS;
+
+    if (!map || !key || (option == EBPF_NOEXIST)) {
+        EBPF_LOG_MESSAGE(EBPF_TRACELOG_LEVEL_ERROR, EBPF_TRACELOG_KEYWORD_MAP, "Invalid argument(s) rejected");
         return EBPF_INVALID_ARGUMENT;
```

Comment:
Working on adding additional tests to address code coverage issues (where possible)

Reply:
done

Reply:
done